Documentation/contributor-guide/triage_issues.md

```diff
@@ -48,9 +48,17 @@ Adding a `type` label to an issue helps create visibility on the health of the p
 
 ### Support requests
 
-Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. In general we prefer that support requests are raised as [discussions](https://github.com/etcd-io/etcd/discussions), however if you identify that an issue is a support request please add the `type/support` or `type/question` label.
+Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. Support requests for etcd should be instead raised as [discussions](https://github.com/etcd-io/etcd/discussions).
```

Comment:
Please provide more exact criteria to identifying support issues. For me this is:
* Questions/configuration/misuse of existing and well documented features. If feature is not well documented we should change the issue type to type/documentation and propose documentation improvements that would prevent future users from stumbling on the problem again.
* Report on etcd outside of support window and/or user is not willing to reproduce the issue on supported and latest minor versions. I want to shift the responsibility to confirm that issue was fixed from contributors to community in discussions.
* User has not provided a proper list of steps to reproduce issue and/or contributors are not able to reproduce the issue. We should limit the effort we put into reproducing issues ourselves and motivate users to provide necessary information to accept the bug report.

Open for other criteria, I just want to make sure we can properly scale up support by avoiding fixing singular issues and finding patterns of problems that users encounter so we can mitigate them.

Reply:
Agree - I've added four common scenarios, with example github issues so that contributors can get an understanding of what these can look like.